docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,984 @@
+  - /tutorials/filter-modular-component
+authors: ["Sky Leilani"]
+languages: []
+viamresources: ["vision", "camera"]
+level: "Beginner"
+date: "2023-09-17"
+# updated: ""
+cost: "0"
+no_list: true
+weight: 3
+---
+
+You can write filter {{< glossary_tooltip term_id="module" text="modules" >}} to selectively store data from your robot based on whether specified conditions have been met.
+For example, you can use a module to capture images based on specific criteria, such as the presence of a color, image quality, or sensor thresholds.
+
+In this tutorial, you will create a [color filter module](https://github.com/viam-labs/modular-filter-examples) and use it to selectively store images from your robot when a specified color is in frame.
+This allows you to take pictures of your pet in the following way:
+
+1. Configure the color filter module to process images and save them only when your pet, along with their easily identifiable colored object, is present in the frame.
+1. Attach a colored object, such as a blue collar, to your pet.
+1. Set up a webcam in an area where your pet is likely to appear in the frame, and set up the color filter-configured camera to periodically capture images and sync them to [Viam's cloud](/services/data/#cloud-sync).
+
+   {{<imgproc src="/tutorials/pet-photographer/data-example.png" resize="550x" declaredimensions=true alt="Dog in blue collar in the camera's live feed">}}
+
+{{< alert title="Note" color="note" >}}
+While this tutorial is about creating a color filter, you can also use it as a reference when writing your own filter module for other components, like a [sensor](https://github.com/viam-labs/modular-filter-examples/tree/main/sensorfilter).
+{{< /alert >}}
+
+## Hardware Requirements
+
+To create your own filtering pet photographer robot, you'll need the following hardware:
+
+- A computer
+- A webcam or an external camera
+- A colored object, such as a blue collar for enhanced accuracy _(optional)_
+
+{{< alert title="Tip" color="tip" >}}
+In this tutorial, the camera is configured to identify and filter images with the color blue, as it is less common in many environments.
+If your pet already has a distinct color that is different from their environment, you can also configure your camera to use that color to identify pictures of your pet.
+{{< /alert >}}
+
+## Set up
+
+Here's how to get started:
+
+1. Install [Go](https://go.dev/dl/) on both your local development computer and on your robot's board if they are not the same device.
+1. [Create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot) and connect to it.
+1. [Install](/installation/manage/#update-viam-server) or [update](/installation/#install-viam-server) `viam-server`.
+   Your `viam-server` must be version 0.8.0 or higher to access the filtering functionality.
+
+## Create or download the filter module
+
+This tutorial will guide you through the process of [coding your filter module](#code-your-entry-point-file).
```

Comment:
Ah, I changed around the organization... so that had been a link to get started coding and skip over the download section, but I don't think it's needed at all since that sections not far down.